site2/website-next/static/reference/next/config/README.md

```diff
@@ -1 +1,28 @@
-> Docs for Pulsar Configuration.
+## Pulsar configurations
+
+Pulsar offers several command-line tools that you can use for managing Pulsar installations, performance testing, using command-line producers and consumers, and more.
+
+You can manage Pulsar configurations through configuration files in the [`conf`](https://github.com/apache/pulsar/tree/master/conf) directory of a Pulsar installation.
+
+- [BookKeeper](/next/config/reference-configuration-bookkeeper/)
+- [Broker](/next/config/reference-configuration-broker/)
+- [Client](/next/config/reference-configuration-client/)
+- [Log4j](/next/config/reference-configuration-log4j/)
+- [Log4j shell](/next/config/reference-configuration-log4j-shell/)
+- [Standalone](/next/config/reference-configuration-standalone/)
+- [WebSocket](/next/config/reference-configuration-websocket/)
+- [Pulsar proxy](/next/config/reference-configuration-pulsar-proxy/)
+- [ZooKeeper](/next/config/reference-configuration-zookeeper/)
```

Comment:
```suggestion
- [BookKeeper](/next/config/reference-configuration-bookkeeper)
- [Broker](/next/config/reference-configuration-broker)
- [Client](/next/config/reference-configuration-client)
- [Log4j](/next/config/reference-configuration-log4j)
- [Log4j shell](/next/config/reference-configuration-log4j-shell)
- [Standalone](/next/config/reference-configuration-standalone)
- [WebSocket](/next/config/reference-configuration-websocket)
- [Pulsar proxy](/next/config/reference-configuration-pulsar-proxy)
- [ZooKeeper](/next/config/reference-configuration-zookeeper)
```

With a trailing slash, the preview failed.

Also, I'm trying to avoid `/next/config` prefix.